repair/row_level.cc

```diff
@@ -2265,12 +2265,20 @@ future<repair_flush_hints_batchlog_response> repair_service::repair_flush_hints_
     co_return repair_flush_hints_batchlog_response();
 }
 
+static locator::host_id get_host_id(const rpc::client_info& cinfo) {
+    const auto* result = cinfo.retrieve_auxiliary_opt<locator::host_id>("host_id");
+    if (!result) {
+        throw std::runtime_error("host_id not found");
```

Comment:
Can you add more context to the message?
Useful info may be:
- call site function name (by adding `seastar::compat::source_location` parameter with `current` default value to `get_host_id`, and using it in the message)
- IP address of `cinfo`

Reply:
dropped the entire commit